packages/backend/src/crawling/worker.ts

```diff
@@ -0,0 +1,110 @@
+import path from "path";
```

Comment:
This file runs in the worker thread (the `workerWrapper.js` is a way of making `ts-node` load the Typescript file in workers.